api/global/internal/noop_propagator.go

```diff
@@ -0,0 +1,36 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/api/propagation"
+)
+
+type NoopPropagator struct {
```

Comment:
I believe that `api/propagation`'s `New()` method with no arguments acts as a no-op propagator. Would it be better to use that?

Reply:
That's a good point, probably worth switching to.

Reply:
will give this a shot. Thanks for your suggestion.